File: HA1_1.py
def Divide(iNo1, iNo2):

    iAns = 0
    iAns = iNo1 / iNo2
    return iAns

def main():
    No1 = 0
    No2 = 0
    Ans = 0

    print("Enter 1st Number")
    No1 = int(input())

    print("Enter 2nd Number")
    No2 = int(input())

    Ans = Divide(No1, No2)

    print(Ans)

if __name__ == "__main__":
    main()

File: HA1_1.cpp
//Program to divide two numbers
#include<iostream>
using namespace std;

int DivideNo(int iNo1, int iNo2)
{
    int iAns = 0;
    if(iNo2 == 0);
    {
        cout<<"Enter a non 0 number"<<"\n";

    }

    iAns = iNo1 / iNo2;

    return iAns;
}

int main()
{
    int iNo1 = 0;
    int iNo2 = 0;
    int iRet = 0;

    cout<<"Enter No1"<<"\n";
    cin>>iNo1;

    cout<<"Enter No2"<<"\n";
    cin>>iNo2;
    
    iRet = DivideNo(iNo1, iNo2);

    cout<<"Ans is %d"<<iRet<<"\n";

    return 0;
}